user_docs/en/changes.t2t

```diff
@@ -83,6 +85,11 @@ What's New in NVDA
 - Fixed a rare case when saving the configuration may fail to save all profiles.  (#16343, @CyrilleB79)
 -In Firefox and Chromium-based browsers,  NVDA will correctly enter focus mode when pressing enter when positioned within a presentational list (ul / ol) inside editable content. (#16325)
 - Column state change is automatically reported when selecting columns to display in Thunderbird message list. (#16323)
+- Fixes for Chromium-based browsers when used with UIA:
+  - Fixed NVDA hang when accessing some UIA documents after they're destroyed (#16393)
+  - Fixed NVDA hang when splitting some UIA text ranges (#16394)
+  - Fixed backspace key not working in GMail sign-in fields when using UIA (#16395)

```

Comment:
A better and less technical wording. Feel free to ignore if you do not find it suitable.

Especially for last item, no need to mention "when using UIA" in the sub-item; we are already in an item dealing with Chromium when used with UIA.

```suggestion
  - Fixed bugs causing NVDA to hang. (#16393, #16394)
  - Backspace key is now working correctly in GMail sign-in fields. (#16395)
```

Reply:
updated as such